presets/ragengine/tests/api/test_main.py

```diff
@@ -48,11 +48,11 @@ async def test_index_documents_success(async_client):
     response = await async_client.post("/index", json=request_data)
     assert response.status_code == 200
     doc1, doc2 = response.json()
-    assert (doc1["text"] == "This is a test document")
+    assert (doc1["text"] == "This is a test document" or
```

Comment:
running `make rag-service-test` locally multiple times causes the original to fail. seems like the indexed arent deleted between tests?